fenix/app/src/test/java/org/mozilla/fenix/home/DefaultSessionControlControllerTest.kt

```diff
@@ -1058,7 +1059,7 @@ class DefaultSessionControlControllerTest {
     @Test
     fun `WHEN private mode button is selected from home THEN handle mode change`() {
```

Comment:
Shouldn't we just restore the old tests from https://github.com/mozilla-mobile/fenix/pull/27262/files#diff-86be192d888c786a8d216844e278ef69e81f0f48e838332d86eadfb9a15c72ea?

The change in this test might have other implications compare to the original tests as seen in the old diff.